pkg/cloud/cloud.go

```diff
@@ -77,7 +77,7 @@ func (f *providerFactory) Get(managedClusterInfo configv1alpha1.ManagedClusterIn
 		return gcp.NewGCPProvider(f.restMapper, f.kubeClient, f.dynamicClient, f.hubKubeClient, eventsRecorder,
 			region, infraID, clusterName, config.Spec.CredentialsSecret.Name,
 			config.Spec.GatewayConfig.GCP.InstanceType, config.Spec.IPSecNATTPort, config.Spec.NATTDiscoveryPort, config.Spec.Gateways)
-	case "Openstack":
+	case "OpenStack":
```

Comment:
This looks like a fundamental bug. Just curious, is this the first attempt where we are validating the end-to-end flow with the RHOS PRs?

Reply:
This was a typo that occurred in the original PR. As I was testing the changes with my local changes it had the correct platform name, but the PR did to have it and was missed in the review.